.github/workflows/ci.yml

```diff
@@ -43,15 +46,47 @@ jobs:
           java -DuseCWDasROOT=true -jar ../.github/kolmafia.jar --CLI _ci_dependencies
         fi
 
-        # Run the verification
-        echo "Verifying..."
+        # Verify autoscend.ash
```

Comment:
You could use an array and loop through the script names, instead of duplicating the code.

Reply:
I don't know how. I am not actually familiar with whatever language it is, and it has necessary yet invisible characters that my editor cannot display (but can copy). I put as a branch on LASS instead of taltamir so that others could contribute code though. So if you know how to switch it to an array you are welcome to commit to this branch.

Reply:
yeah i can do that